api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3053,8 +3088,9 @@ message OneOf {
     events.KubernetesClusterDelete KubernetesClusterDelete = 98;
     events.SSMRun SSMRun = 99;
     events.ElasticsearchRequest ElasticsearchRequest = 100;
-    events.DesktopSharedDirectoryRead DesktopSharedDirectoryRead = 101;
-    events.DesktopSharedDirectoryWrite DesktopSharedDirectoryWrite = 102;
+    events.DesktopSharedDirectoryStart DesktopSharedDirectoryStart = 101;
+    events.DesktopSharedDirectoryRead DesktopSharedDirectoryRead = 102;
```

Comment:
Did read/write get missed in a previous PR? I see the description on this one is only for start.

Reply:
This is being merged into `isaiah/sd-read-write-audit-events`, which is the branch for the open PR that contains read/write:

https://github.com/gravitational/teleport/pull/17348

(Probably best to review that one first and then come take another look at this one).